Wikipedia/UI-V5/TableOfContentsReadMoreItem.swift

```diff
@@ -13,7 +13,15 @@ public protocol TableOfContentsFooterItem : TableOfContentsItem {
 }
 
 public class TableOfContentsReadMoreItem : NSObject, TableOfContentsFooterItem {
-    public let titleText: String = localizedStringForKeyFallingBackOnEnglish("article-read-more-title")
+    let site:MWKSite
+    init(site: MWKSite) {
+        self.site = site
```

Comment:
Fixed. Interestingly in Swift instance vars need to be set before super init... http://stackoverflow.com/a/24150540